spec/features/admin/tags_spec.rb

```diff
@@ -3,16 +3,21 @@
 feature 'Admin tags' do
 
   background do
-    @tag1 = create(:tag)
+    @tag1 = create(:category)
     login_as(create(:administrator).user)
   end
 
   scenario 'Index' do
     create(:debate, tag_list: 'supertag')
+    debate_tag = ActsAsTaggableOn::Tag.find_by(name: "supertag")
     visit admin_tags_path
 
     expect(page).to have_content @tag1.name
-    expect(page).to have_content 'supertag'
+    if debate_tag.category?
```

Comment:
Well, I must admit that I've made a weird test. In background of this spec `@tag1` is created, a tag of kind "category". This tag is expected to be managed in admin/tags controller. In the "Index" scenario another tag named "supertag" is created, associated with a debate. Before the changes of this branch, the test was expecting to find also "supertag" in the index list. But now, the "supertag" won't be listed since the debate's tags are created with `kind` attribute as `nil`, not "category". I'm not sure if in the futute the creation of tags associated with debates or other taggable models should be changed to include the "category" kind by default, so I've tried to prepare the test to both cases: when the tags of a new debate are categories and when not. 

I think I made things more complicated. I'll simplify it and make it clearer.